src/frontend/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+//import "../App.css";
+import "./SearchBar.css";
+
+// Start array of user objects
+const users = [
+  {
+    id: 1,
+    name: 'Anitha Dharneedharan ',
+    Language: 'JavaScript',
+    Location: 'Sutton'
+  },
+  {
+    id: 2,
+    name: 'Florin Dumitru',
+    Language: 'Python',
+    Location: 'Walthamstow'
+  },
+  {
+    id: 3,
+    name: 'Kainy Ryu',
+    Language: 'Java',
+    Location: 'Bethnal Green'
+  },
+  {
+    id: 4,
+    name: 'Kamila Lew',
+    Language: 'C++',
+    Location: 'Heathrow'
+  },
+  {
+    id: 5,
+    name: 'Selchuk Karakus',
+    Language: 'Ruby',
+    Location: 'Leytonstone'
+  }
+];
+
+// End of user objects
```

Comment:
I think it'll be better to have a js file for the data

Reply:
To complete @KainyRyu I think this should be moved into a JSON file. It might be this was the file type that @KainyRyu  was referring to?